catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -719,7 +719,15 @@ protected InputTransformer lookupTransformerReference(String namespaceUri)
       Collection<ServiceReference<InputTransformer>> transformerReference =
           bundleContext.getServiceReferences(
               InputTransformer.class, "(schema=" + namespaceUri + ")");
-      return bundleContext.getService(transformerReference.iterator().next());
+      if (transformerReference.isEmpty()) {
```

Comment:
✏️  Cyclomatic complexity can be reduced. Invert this `if` statement and `return` early so there's no need for an `else`.